instance_skel.d.ts

```diff
@@ -60,7 +61,11 @@ declare abstract class InstanceSkel<TConfig> {
 	 * Processes a feedback state.
 	 * @since 1.0.0
 	 */
-	feedback?(feedback: CompanionFeedbackEvent, info: CompanionFeedbackEventInfo): CompanionFeedbackResult
+	feedback?(
```

Comment:
Is there a reason this isn't in the types files and I see a duplicate in there?

Reply:
I dont understand what you mean?